crates/services/p2p/src/behavior.rs

```diff
@@ -35,6 +39,10 @@ use libp2p::{
     PeerId,
 };
 
+const MAX_PENDING_INCOMING_CONNECTIONS: u32 = 100;
+const MAX_PENDING_OUTGOING_CONNECTIONS: u32 = 100;
+const MAX_ESTABLISHED_CONNECTIONS: u32 = 1000;
```

Comment:
Should we add configuration variables for these constants as well? I'm also not completely following the difference between `MAX_ESTABLISHED_CONNECTIONS` and the existing configuration values. Shouldn't this just be the sum of the `max_(discovery|gossipsub|request_response)_peers_connected` parameters?

Reply:
nope, my reasoning for keeping them non-configurable is to prevent footguns by the node operator